.github/workflows/cargo.yml

```diff
@@ -3,68 +3,59 @@ name: cargo
 on: [pull_request]
 
 jobs:
-  # Ensure all Rust code is properly formatted with `rustfmt`.
+  # Enforce coding style
   cargo-fmt:
     runs-on: ubuntu-latest
     container: quay.io/enarx/fedora-test
     steps:
-    - uses: actions/checkout@v2
-    
-    - name: rustfmt
-      run: cargo fmt -- --check
+      - uses: actions/checkout@v2
+      - name: Code is formatted correctly.
+        run: cargo make check-format
 
-  # Build the Enarx workspace.
-  cargo-build:
+  # Verify compatible licensing.
+  cargo-deny:
     runs-on: ubuntu-latest
     container: quay.io/enarx/fedora-test
     steps:
-    - uses: actions/checkout@v2
-    
-    # Ensure the project builds correctly.
-    - name: Build
-      run: cargo build
+      - uses: actions/checkout@v2
+      - name: Dependencies are license-compatible.
+        run: cargo make deny
 
-  # Test the Enarx workspace.
-  cargo-test:
+  # Scan database for reported vulnerabilities.
+  cargo-audit:
     runs-on: ubuntu-latest
     container: quay.io/enarx/fedora-test
     steps:
-    - uses: actions/checkout@v2
-    
-    # Run any specified Rust tests.
-    - name: Run tests
-      run: cargo test
+      - uses: actions/checkout@v2
+      - name: No unresolved reported vulnerabilities.
+        run: cargo make audit
 
-    # Run Rust tests with OpenSSL-specific features enabled for SEV.
-    - name: Run tests (SEV-specific)
-      run: cargo test --manifest-path sev/Cargo.toml --verbose --features=openssl
```

Comment:
This test needs to be added back in.

Reply:
It's not added back in as a separate run step under GitHub actions, but the `Makefile.toml` inside the `sev` crate extends the `cargo test` target to enable the feature here: https://github.com/enarx/enarx/pull/420/files#diff-8ed61aa622ed1d7f91c707e0fd48d01d